packages/smartcontract/contracts/JanusNameService.sol

```diff
@@ -26,21 +27,22 @@ contract JanusNameService {
         string storageHash;
         uint expires;
         address owner;
+        uint ownerIndex;
     }
 
-    TopDomain[] private topDomains;
```

Comment:
why did you change the topDomains array and mappings to public?

Reply:
It helped me debug the problem we had with updating the indexes for ownership transfer.
Furthermore, I added some regression tests that depend on this property to be public.

Reply:
yeah, i forgot about the tests.
Make sense.